pkg/apis/provisioning/v1alpha1/provisioner.go

```diff
@@ -106,8 +106,10 @@ var (
 	// Reserved labels
 	ProvisionerNameLabelKey      = SchemeGroupVersion.Group + "/name"
 	ProvisionerNamespaceLabelKey = SchemeGroupVersion.Group + "/namespace"
-	ProvisionerUnderutilizedKey  = SchemeGroupVersion.Group + "/underutilized"
-	ProvisionerTTLKey            = SchemeGroupVersion.Group + "/ttl"
+	//ProvisionerUnderutilizedKey  = SchemeGroupVersion.Group + "/underutilized"
+	ProvisionerTTLKey = SchemeGroupVersion.Group + "/ttl"
```

Comment:
Can you move this outside of the labels, since it's an annotation? Maybe it should belong in the utilization controller code.